docker-compose.yaml

```
@@ -0,0 +1,120 @@
+version: "3.7"
+
+services:
+
+  #databases
+  setup:
+    container_name: setup
+    image: ghcr.io/gamify-it/test-data:latest
+    pull_policy: always
+    restart: on-failure
+    environment:
+      POSTGRES_SETUP: >
+        keycloak/default.sql
+        overworld/default.sql
+
+  keycloak-db:
+    container_name: keycloak-db
+    image: postgres:14-alpine
+    restart: always
+    expose:
+      - "5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+
+  fileserver-db:
+    container_name: fileserver-db
+    image: postgres:14-alpine
+    restart: always
+    expose:
+      - "5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+
+  overworld-db:
+    container_name: overworld-db
+    image: postgres:14-alpine
+    restart: always
+    expose:
+      - "5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+
+  #backends
+  keycloak:
+    container_name: keycloak
+    expose:
+      - "80"
+    image: quay.io/keycloak/keycloak:19.0
+    command:
+      - "start-dev"
+    environment:
+      KEYCLOAK_ADMIN: admin
+      KEYCLOAK_ADMIN_PASSWORD: admin
+
+      KC_DB: postgres
+      KC_DB_URL: jdbc:postgresql://keycloak-db:5432/postgres
+      KC_DB_USERNAME: postgres
+      KC_DB_PASSWORD: postgres
+
+      KC_HOSTNAME_URL: http://localhost/keycloak
+      KC_HOSTNAME_STRICT: false
+      KC_PROXY: edge
+      KC_HTTP_RELATIVE_PATH: /keycloak
+      KC_HTTP_PORT: 80
+
+  fileserver:
+    container_name: fileserver
+    build: .
+    restart: always
+    expose:
+      - "80"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_URL=postgresql://fileserver-db:5432/postgres
+      - BASE_URL=http://localhost
+      - KEYCLOAK_CLIENT_ID=fileserver
+      - KEYCLOAK_CLIENT_SECRET=test-client-secret
+      - KEYCLOAK_ISSUER=http://localhost/keycloak/realms/Gamify-IT
+      - KEYCLOAK_URL=http://keycloak/keycloak/realms/Gamify-IT
+
+  overworld-backend:
+    container_name:   overworld-backend
+    image: ghcr.io/gamify-it/overworld-backend
+    restart: always
+    pull_policy: always
+    expose:
+      - "80"
+    depends_on:
+      - overworld-db
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_URL=postgresql://overworld-db:5432/postgres
+      - MEMORY_URL=http://memory-backend/api/v1
+      - KEYCLOAK_ISSUER=http://localhost/keycloak/realms/Gamify-IT
+      - KEYCLOAK_URL=http://keycloak/keycloak/realms/Gamify-IT
+
+
+  #frontends
+  landing-page:
+    container_name: landing-page
+    image: ghcr.io/gamify-it/landing-page:main
+    restart: always
+    pull_policy: always
+    expose:
+      - "80"
+
+  # reverse-proxy
+  reverse-proxy:
+    container_name: reverse-proxy
+    image: nginx
+    restart: always
+    volumes:
+      - ./.nginx/compose/nginx.conf:/etc/nginx/conf.d/default.conf:ro
+    ports:
+      - "80:80"
```

Newline at the end makes the file look cleaner.
```suggestion
      - "80:80"

```